packages/amplify-category-api/src/graphql-transformer/override.ts

```diff
@@ -63,26 +62,17 @@ export function applyFileBasedOverride(stackManager: StackManager, overrideDirPa
   });
 
   const appsyncResourceObj = convertToAppsyncResourceObj(amplifyApiObj);
-  const overrideCode: string = fs.readFileSync(overrideFilePath, "utf-8");
-  const sandboxNode = new vm.NodeVM({
-    console: "inherit",
-    timeout: 5000,
-    sandbox: {},
-    require: {
-      context: "sandbox",
-      builtin: ["path"],
-      external: true
-    }
-  });
-  // Remove these when moving override up to amplify-category-api level
   const { envName } = stateManager.getLocalEnvInfo();
   const { projectName } = stateManager.getProjectConfig();
   const projectInfo = {
     envName,
     projectName
   };
   try {
-    sandboxNode.run(overrideCode, overrideFilePath).override(appsyncResourceObj, projectInfo);
+    const overrideImport = require(overrideFilePath);
```

Comment:
Since this is a synchronous method, `await import` doesn't work. Have to fall back to `require`.